lib/mix/test/mix/tasks/xref_test.exs

```diff
@@ -1127,6 +1209,68 @@ defmodule Mix.Tasks.XrefTest do
       """)
     end
 
+    test "group with sink (json)" do
+      assert_graph_json(
+        ~w[--group lib/a.ex,lib/c.ex,lib/e.ex --sink lib/e.ex],
+        """
+        { "lib/a.ex+": { "lib/b.ex":  "compile",
+                         "lib/d.ex":  "compile" },
+          "lib/b.ex":  { "lib/a.ex+": "compile" },
+          "lib/d.ex":  { "lib/a.ex+": "runtime" } }
+        """,
+        strip_ws: true
+      )
+    end
+
+    @triple_cycle %{
+      "lib/a.ex" => """
+      defmodule A do
+        def a, do: :ok
+        B.b2()
+      end
+      """,
+      "lib/b.ex" => """
+      defmodule B do
+        def b1, do: C.c()
+        def b2, do: :ok
+      end
+      """,
+      "lib/c.ex" => """
+      defmodule C do
+        def c, do: A.a
+      end
+      """
+    }
+    test "triple cycle (dot)" do
```

Comment:
Let's keep this test and the one below but I think we can just use the `@default_files`. It already has a cycle in it, which is the important thing to test (i.e. we make sure that the formatting does not loop). WDYT?

Reply:
done.